sdk/servicebus/service-bus/src/serviceBusRuleManager.ts

```diff
@@ -190,7 +190,7 @@ export class ServiceBusRuleManagerImpl implements ServiceBusRuleManager {
     do {
       const rules = await this.getRules({
         skip: Number(marker),
-        maxCount: options.maxPageSize,
+        maxCount: options.maxPageSize ?? 100,
```

Comment:
if we don't set it, the internal managementClient will use max_int as maxCound. Plus I want to use 100 as default page size as well like .NET does

Reply:
We could change the management client too. I haven't tested whether it would work if maxCount is not set.

Reply:
> We could change the management client too

I would vote to do that because I don't think we should have our own defaults and the service should use the best default

Reply:
I think this is the right place to control paging size, as `managementClient.getRules()` is not a page operation, if you call it without maxCount it by default will retrieve all.

Reply:
https://github.com/Azure/azure-sdk-for-js/blob/main/sdk/servicebus/service-bus/src/core/managementClient.ts#L1141-L1157

Reply:
Oh I see,  yeah if `getRules` does not return pages, then the change here makes sense. Might be a good idea to agree on the default across all languages and the service.

Reply:
Sure. And I need to check whether the ATOM based admin client has the same issue.